src/main/webapp/aboutus.html

```diff
@@ -47,11 +47,11 @@ <h2>Teammate C Name</h2>
       <li>Aspirational Hobby: </li>
       <li>Ask me About: </li>
     </ul>
-    <h2>Teammate D Name (delete if you team only has 3 students) </h2>
```

Comment:
spelling error, change "you" to "your"

Reply:
Good catch! That typo was actually in the original repo: I deleted this whole line and changed it to just say "Kevin" instead.

But this comment helped me spot another typo in my code, I accidentally typed "expecially" instead of "especially". I've fixed that!